tests/annotation_model_tests/test_annotation_model_nieuwkoopse_plassen.py

```diff
@@ -0,0 +1,192 @@
+import pickle
+import glob
+import rasterio
+import numpy as np
+import pandas as pd
+import contextlib
+from tif_model_iterator import tif_kernel_iterator
```

Comment:
This and the import below will fail when the rename of the other repo comes through.